ADAL/tests/XCTestCase+TestHelperMethods.m

```diff
@@ -501,5 +496,136 @@ - (ADTestURLResponse *)adResponseAuthCode:(NSString *)authCode
     return response;
 }
 
+- (MSIDTokenCacheItem *)adCreateAccessMSIDTokenCacheItem
+{
+    MSIDTokenCacheItem *tokenCacheItem = [MSIDTokenCacheItem new];
+    tokenCacheItem.accessToken = TEST_ACCESS_TOKEN_TYPE;
+    tokenCacheItem.refreshToken = nil;
+    tokenCacheItem.idToken = [self adCreateUserInformation:TEST_USER_ID].rawIdToken;
```

Comment:
nit: we have utilities to create ID token in MSID. Let's try to switch to MSID, so we can get rid of ADAL utilities faster.

Reply:
done